internal/constellation/kubecmd/kubecmd_test.go

```diff
@@ -611,6 +620,62 @@ func TestApplyJoinConfig(t *testing.T) {
 	}
 }
 
+func TestRetryAction(t *testing.T) {
+	maxRetries := 3
+
+	testCases := map[string]struct {
+		failures int
+		wantErr  bool
+	}{
+		"no failures": {
+			failures: 0,
+		},
+		"fail once": {
+			failures: 1,
+		},
+		"fail equal to maxRetries": {
+			failures: maxRetries,
+			wantErr:  true,
+		},
```

Comment:
I would have expected this to succeed - is `maxRetries` actually `maxAttempts`?

Reply:
yes